lxd/storage/drivers/driver_btrfs.go

```diff
@@ -285,7 +285,7 @@ func (d *btrfs) Validate(config map[string]string) error {
 		"btrfs.mount_options": validate.IsAny,
 	}
 
-	return d.validatePool(config, rules)
+	return d.validatePool(config, rules, nil)
```

Comment:
Can you separate the commit that changes the validation from the commit that changes the default values please, for clarity?